bai2.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<script>
+  for(i=1;i<=20;i++){
```

Comment:
Thường 1 vòng lặp for bắt đầu vs index = 0, vậy làm thế nào in ra số bắt đầu từ 1 thì nên thử làm for(i=0,...) để hiểu thêm.